test/e2e/apimachinery/resource_quota.go

```diff
@@ -915,6 +918,57 @@ var _ = SIGDescribe("ResourceQuota", func() {
 			framework.Failf("Expected `not found` error, got: %v", err)
 		}
 	})
+
+	ginkgo.It("should manage the lifecycle of a ResourceQuota", func() {
+		client := f.ClientSet
+		ns := f.Namespace.Name
+
+		rqName := "e2e-quota-" + utilrand.String(5)
+		label := map[string]string{"e2e-rq-label": rqName}
+		labelSelector := labels.SelectorFromSet(label).String()
+
+		ginkgo.By("Creating a ResourceQuota")
+		resourceQuota := &v1.ResourceQuota{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   rqName,
+				Labels: label,
+			},
+			Spec: v1.ResourceQuotaSpec{
+				Hard: v1.ResourceList{},
+			},
+		}
+		resourceQuota.Spec.Hard[v1.ResourceCPU] = resource.MustParse("1")
+		resourceQuota.Spec.Hard[v1.ResourceMemory] = resource.MustParse("500Mi")
+		_, err := createResourceQuota(client, ns, resourceQuota)
+		framework.ExpectNoError(err)
+
+		ginkgo.By("Getting a ResourceQuota")
+		resourceQuotaResult, err := client.CoreV1().ResourceQuotas(ns).Get(context.TODO(), rqName, metav1.GetOptions{})
+		framework.ExpectNoError(err)
+		framework.ExpectEqual(resourceQuotaResult.Spec.Hard[v1.ResourceCPU], resource.MustParse("1"))
+		framework.ExpectEqual(resourceQuotaResult.Spec.Hard[v1.ResourceMemory], resource.MustParse("500Mi"))
+
+		ginkgo.By("Listing all ResourceQuotas with LabelSelector")
+		rq, err := client.CoreV1().ResourceQuotas("").List(context.TODO(), metav1.ListOptions{LabelSelector: labelSelector})
+		framework.ExpectNoError(err, "Failed to list job. %v", err)
+		framework.ExpectEqual(len(rq.Items), 1, "Failed to find ResourceQuotes %v", rqName)
+
+		ginkgo.By("Patching the ResourceQuota")
+		payload := "{\"metadata\":{\"labels\":{\"" + rqName + "\":\"patched\"}}}"
```

Comment:
I might suggest also patching a resource quota spec value here.  ResourceQuota is a "built-in" object, and changing a quota is a part of conformance as well.

Reply:
The payload for the patch now includes a change to the spec

https://github.com/kubernetes/kubernetes/blob/d1447d6419c330224594530b5ed969466b4def83/test/e2e/apimachinery/resource_quota.go#L957

Included a check to confirm the change to the spec

https://github.com/kubernetes/kubernetes/blob/d1447d6419c330224594530b5ed969466b4def83/test/e2e/apimachinery/resource_quota.go#L961